### DISCUSSION TITLE
[Markdown editor] Numbered/Unordered/Task list broken?

### DISCUSSION THREAD

**Discussion by marcelolx** [community/community#73168] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Before, I could create nested lists by just typing Tab on the next line of the previous element of the list, now, it seems that this is gone? At least I don't see a mention of that in the docs https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#nested-lists

I guess there are reasons for it, but the experience, IMO, got worse because now you press TAB, and it goes to the "Create pull request" button instead of creating a nested item... I don't want to press space multiple times to create a nested list.

Also, I have noticed that the tab nav/panel border is not aligned with the markdown editor 
![image](https://github.com/community/community/assets/20196775/0a9acaaf-6618-468c-99cb-676882aec8ef)

**Comment by drago1520** [comment]
mhh, same problem
